cmd/acra-translator/grpc_api/decryptor_test.go

```diff
@@ -0,0 +1,253 @@
+package grpc_api
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"github.com/cossacklabs/acra/cmd/acra-translator/common"
+	"github.com/cossacklabs/acra/network"
+	"github.com/cossacklabs/acra/utils/tests"
+	"github.com/cossacklabs/themis/gothemis/keys"
+	"github.com/stretchr/testify/assert"
+	"go.opencensus.io/trace"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/status"
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+)
+
+const connectionTimeout = time.Second
+
+// newTestKeystore return keystore implementation with generated keypairs for encryption/decryption and poison record purposes
+func newTestKeystore(t *testing.T) *testKeystore {
+	keypair, err := keys.New(keys.TypeEC)
+	if err != nil {
+		t.Fatal(err)
+	}
+	poisonKeypair, err := keys.New(keys.TypeEC)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return &testKeystore{EncryptionKeypair: keypair, PoisonKey: poisonKeypair}
+}
+
+// testgRPCServer start gRPC server as gorountine using unix socket
+type testgRPCServer struct {
+	listener net.Listener
+	server   *grpc.Server
+	unixPath string
+}
+
+func newServer(data *common.TranslatorData, opts []grpc.ServerOption, t *testing.T) *testgRPCServer {
+	factory := &GRPCServerFactory{}
+	server, err := factory.New(data, opts...)
+	if err != nil {
+		t.Fatal(err)
+	}
+	unixPath, err := ioutil.TempFile("", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	os.Remove(unixPath.Name())
+	listener, err := net.Listen("unix", unixPath.Name())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	go server.Serve(listener)
+	return &testgRPCServer{listener: listener, server: server, unixPath: unixPath.Name()}
+}
+
+// newServerTLSgRPCOpts returns server options to use TLS as transport using keys from tests/ssl/[ca|acra-server]
+func newServerTLSgRPCOpts(t *testing.T) []grpc.ServerOption {
+	verifier := network.NewCertVerifierAll()
+	workingDirectory := tests.GetSourceRootDirectory(t)
+	serverConfig, err := network.NewTLSConfig("localhost", filepath.Join(workingDirectory, "tests/ssl/ca/ca.crt"), filepath.Join(workingDirectory, "tests/ssl/acra-server/acra-server.key"), filepath.Join(workingDirectory, "tests/ssl/acra-server/acra-server.crt"), 4, verifier)
```

Comment:
Such a long line, don't you want to split it, one arg per line, for the sake of readability?